tests/perf/context_times.sh

```diff
@@ -17,13 +17,13 @@ get_context_time() {
 }
 
 log_this_revision() {
-    rev=`git log -1 --pretty="format:%h"`
-    printf $rev
-    title=`git log -1 --pretty="format:%s"`
-    printf ",\"$title\""
-    for i in `seq 1 5`; do
-        rv=`$1`
-        printf ",$rv"
+    rev=$(git log -1 --pretty="format:%h")
+    printf "%s" "$rev"
+    title=$(git log -1 --pretty="format:%s")
+    printf ",\"%s\"" "$title"
+    for _ in $(seq 1 5); do
```

Comment:
a little confused about this TBH

Reply:
This just collects 5 samples

Reply:
yes, but isn't this doing 5 times the same thing?

Reply:
ah, nvm, next time i should try and understand the code before asking :)